search-app/src/cmr/search/data/metadata_retrieval/metadata_transformer.clj

```diff
@@ -147,19 +156,29 @@
 (defmethod transform-with-strategy :migrate-umm-json
   [context concept _ target-formats]
   (let [{concept-mime-type :format, metadata :metadata, concept-type :concept-type} concept
-        source-version (umm-spec/umm-json-version concept-type concept-mime-type)]
-    (reduce (fn [translated-map target-format]
-              (assoc translated-map target-format
-                     (umm-json/umm->json
-                       (u/remove-nils-empty-maps-seqs
-                         (vm/migrate-umm context
-                                         concept-type
-                                         source-version
-                                         (umm-spec/umm-json-version concept-type
-                                                                    target-format)
-                                         (json/decode metadata true))))))
-            {}
-            target-formats)))
+        source-version (umm-spec/umm-json-version concept-type concept-mime-type)
+        [t result] (u/time-execution (reduce (fn [translated-map target-format]
+                                               (assoc translated-map target-format
+                                                      (umm-json/umm->json
+                                                       (u/remove-nils-empty-maps-seqs
+                                                        (vm/migrate-umm context
+                                                                        concept-type
+                                                                        source-version
+                                                                        (umm-spec/umm-json-version concept-type
+                                                                                                   target-format)
+                                                                        (json/decode metadata true))))))
+                                             {}
+                                             target-formats))]
+    (info "transform-with-strategy migrate-umm-json: "
+          "time: " t
+          "concept-mime-type: " concept-mime-type
+          "concept-type: " concept-type
+          "parent request num-concepts: " (:num-concepts concept)
+          "target-formats: " target-formats
+          "source version: " source-version
+          "provider: " (:provider-id concept)
+          "metadata length: " (count metadata))
```

Comment:
Same question as above. Are we really showing metadata length or count? Can we rename this filed "number of x in metadata" if it's counting something specific?

Reply:
this is one record, but we want to know how large the file is to see if bigger metadata records are always longer to process. It is a poor proxy for complexity of metadata.